test/parallel/test-util-inspect.js

```diff
@@ -147,8 +147,8 @@ assert.strictEqual(
   "[String: 'hello'] { [length]: 5, [Symbol(foo)]: 123 }"
 );
 
-assert.strictEqual(util.inspect((new JSStream())._externalStream),
-                   '[External]');
+assert.match(util.inspect((new JSStream())._externalStream),
+             /^\[External: .*?\]$/);
```

Comment:
Can you check that the value is a non-empty hex string?

Reply:
```suggestion
             /^\[External: [0-9a-f]+\]$/);
```